### PR TITLE
Feature/1. create nav bar

### PR DESCRIPTION
**Pull request by huymichael**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+<head>
+    <title>HTML FIRST TIME</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <div class="page">
+        <header class="header">
+            <nav>
```
Comment (techcoaching) [inline index.html]: I think we did not need nav here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <a class="brand" href="#" title="homepage">
+                    <img src="logo.png" alt="companybrand" class="icon"/>
+                    <span class="companyname">Perchu</span>
```
Comment (techcoaching) [inline index.html]: I think we should  change the name to 'company-name' as convention for css

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                </a>
+                <div class="menu">
```
Comment (techcoaching) [inline index.html]: menu, we can have more than 1, please give the name more specifically, such as main-menu. as we only have 1 main-menu in the app :)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                    <ul class="navbar-nav">
```
Comment (techcoaching) [inline index.html]: could you tell me what does "navbar-nav" mean in your app? 
Please named the component with meaningful word in your context

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+                        <li class="active">
+                            <a href="#" title="home"> Home</a>
+                        </li>
+                        <li>
```
Comment (techcoaching) [inline index.html]: for normal li, We did not have css class for them, so we can not style those

File: style.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    background-color: pink;
+}
+
+.page .footer {
+    height: 100px;
+    background-color: orange;
+}
+/* end of page */
+
+/* navigation */
```
Comment (techcoaching) [inline style.css]: each component should be located in its own region please

File: style.css [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+.brand {
+    float: left;
```
Comment (techcoaching) [inline style.css]: with this, we will got trouble if move this component to other container and expect the component (branch) right align)

File: style.css [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    margin-top: 20px;
+    margin-left: 30px;
+    margin-right: 5px;
+}
+
+.brand .companyname {
+    font-size: 30px;
+}
+
+.menu {
```
Comment (techcoaching) [inline style.css]: is there any reason that menu and branch were grouped into the same region??

File: style.css [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+    float: right;
+    font-size: 20px;
+    text-align: center;
+    white-space: nowrap;
+    overflow: hidden;
+    line-height: 38px;
+    height: 40px;
+}
+
+.navbar-nav li {
```
Comment (techcoaching) [inline style.css]: name of html tag was not allowed in css file

File: style.css [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+    display: inline;
+    text-align: center;
+    padding: 14px 16px;
+}
+
+.navbar-nav li a {
```
Comment (techcoaching) [inline style.css]: a can not be used in css file

File: style.css [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+    color: blue;
+    text-decoration: none;
+}
+
+.active {
```
Comment (techcoaching) [inline style.css]: .active can not be a stand along css class.

**Comment by huymichael** [comment]
ăn hành

**Pull request closed without merging**